appveyor.yml

```diff
@@ -1,33 +1,29 @@
-# appveyor file
-# http://www.appveyor.com/docs/appveyor-yml
-
-init:
-  - git config --global core.autocrlf input
-
 branches:
   only:
     - master
-
-# what combinations to test
+init:
+  - git config --global core.autocrlf input
 environment:
   matrix:
-    - nodejs_version: 7
+    - nodejs_version: '8'
+      webpack_version: 3.0.0
       job_part: test
-    - nodejs_version: 6
+    - nodejs_version: '6'
+      webpack_version: 3.0.0
       job_part: test
-    - nodejs_version: 4
+    - nodejs_version: '4.3'
+      webpack_version: 3.0.0
```

Comment:
Flip the version to `latest` from `3.0.0` 

&& 

change `cmd: npm install webpack@^%webpack_version%`

to 

`cmd: npm install webpack@%webpack_version%`

This needs to be updated in defaults